docs/snapshotting/snapshot-editor.md

```diff
@@ -0,0 +1,59 @@
+# Snapshot editor
+
+The `snapshot-editor` is a program for modification of Firecracker snapshots.
+
+## Prior knowledge
+
+Firecracker snapshot consists of 2 files:
+
+- `vmstate` file: file with Firecracker internal data such as vcpu states,
+  devices states etc.
+- `memory` file: file with guest memory.
+
+`snapshot-editor` (currently) allows to modify only `vmstate`files only
+on aarch64 platform.
+
+## Usage
+
+### `edit-vmstate` command
+
+#### `remove-regs` subcommand (aarch64 only)
+
+This command is used to remove specified registers from vcpu states inside
+vmstate snapshot file.
+
+Usage:
+
+```bash
+snapshot-editor edit-vmstate remove-regs \
+    --vmstate-path <VMSTATE_PATH> \
+    --output-path <OUTPUT_PATH> \
+    [REGS]...
```

Comment:
Here, as well, we should describe what type of things are `REGS`

Reply:
Added parameters descriptions